### PR TITLE
README for running and testing - BackEnd

### PR DESCRIPTION
**Pull request by dylanlaffin**
The readme file updated to suit our project

### CODE REVIEW

**Review by lexseaton — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -49,13 +49,38 @@ How to start the test application
...
     3. DB_HOST
     4. DB_NAME
-1. Run `mvn clean install` to build your application
-1. You can start application via:
+2. Run `mvn clean install` to build your application
+3. You can start application via:
     1. Terminal: `java -jar target/java-swagger-flyway-starter-org.kainos.ea.jar server config.yml`
     2. IDE: Edit run configuration -> Add `server` to program arguments -> Run
-1. To check that your application is running enter url `http://localhost:8080/api/test`
+4. To check that your application is running enter url `http://localhost:3000`
```
Comment (boydstepkainos) [inline README.md]: Add a valid path for the project to the URL

**Review by boydstepkainos — DISMISSED** [review]
There's sections left blank, such as technology. Please populate

File: README.md [hunk 2]
```diff
@@ -49,13 +49,38 @@ How to start the test application
...
+
+Project Description
+---
+An online job application that serves both,
+Kainos recruitment admin to retrieve and update job roles,
+and their relevant information and applicants to apply for roles.
+
+Technology Stack
+---
+Front-End: IntelliJ, HTML, Docker, swagger, Css, Javascript
```
Comment (boydstepkainos) [inline README.md]: Don't need to list IDE's

File: README.md [hunk 3]
```diff
@@ -49,13 +49,38 @@ How to start the test application
...
     3. DB_HOST
     4. DB_NAME
-1. Run `mvn clean install` to build your application
-1. You can start application via:
+2. Run `mvn clean install` to build your application
+3. You can start application via:
     1. Terminal: `java -jar target/java-swagger-flyway-starter-org.kainos.ea.jar server config.yml`
     2. IDE: Edit run configuration -> Add `server` to program arguments -> Run
-1. To check that your application is running enter url `http://localhost:8080/api/test`
+4. To check that your application is running enter url `http://localhost:3000/loginForm`
```
Comment (shaunganley) [inline README.md]: this should be pointing to an api url

File: README.md [hunk 4]
```diff
@@ -49,13 +49,38 @@ How to start the test application
...
+
+Project Description
+---
+An online job application that serves both,
+Kainos recruitment admin to retrieve and update job roles,
+and their relevant information and applicants to apply for roles.
+
+Technology Stack
+---
+Front-End: HTML, Docker, swagger, Css, Javascript
```
Comment (shaunganley) [inline README.md]: no need for frontend info in the api

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**